src/components/Comments/Comments.module.css

```diff
@@ -24,146 +21,236 @@
   align-items: center;
   gap: 32px;
 }
-.commentsBox {
+
+.leftSvg,
+.rightSvg,
+.rightSvgLandingPage,
+.leftSvgLandingPage {
+  display: none;
+}
+
+.commentsBox,
+.commentsBoxLandingPage {
   background-color: var(--primary-background-color);
   border-radius: 0px 64px;
   display: flex;
   justify-content: center;
   align-items: center;
-  padding-top: 27px;
-  padding-bottom: 37px;
-  padding-left: 37px;
-  padding-right: 31px;
+  padding: 30px;
 }
-.commentsText {
+.commentsText,
+.commentsTextLandingPage {
   font-family: var(--main-font);
-  font-size: 16px;
-  font-style: normal;
-  font-weight: 400;
-  line-height: 24px;
-  letter-spacing: 0.019px;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  letter-spacing: 0.00119rem;
 }
 
-@media screen and (min-width: 394px) {
+@media screen and (min-width: 835px) {
```

Comment:
A nie mieliśmy przypadkiem użyć @media screen and (min-width: 765px) dla rozdzielczości tabletowych?

Reply:
to było zanim przyklepaliśmy :D

Reply:
Ok :D To wrzucaj commita z poprawkami i mergujemy :D